src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs

```diff
@@ -58,9 +58,22 @@ private bool ShouldShowLightDismissOverlay
 
 		public Popup()
 		{
+			this.RegisterDefaultValueProvider(GetDefaultValue);
 			Initialize();
 		}
 
+		private bool GetDefaultValue(DependencyProperty dependencyProperty, out object value)
```

Comment:
Unrelated to your changes: Was there any reason to use `object` instead of `bool?` for the out parameter? Or more simply, eliminate the out parameter and return `bool?`

This doesn't seem to be publicly exposed, so it could be changed without any breaking changes.

Reply:
The `GetDefaultValue` matches the `DefaultValueProvider` delegate and is called for all dependency properties - so it has to be `object` to accommodate for any type of DP. I think for example `Control` class has some provider as well, which handles focus-related properties, etc.